scripts/check_doc_ipfs_hashes.py

```diff
@@ -232,11 +236,13 @@ def check_ipfs_hashes(  # pylint: disable=too-many-locals,too-many-statements
     # Fix full commands on docs
     for md_file in all_md_files:
         content = read_file(str(md_file))
-        for match in re.findall(AEA_COMMAND_REGEX, content):
+        for match in [m.groupdict() for m in re.finditer(AEA_COMMAND_REGEX, content)]:
             matches += 1
-            doc_full_cmd = match[0]
-            doc_cmd = match[2]
-            doc_hash = match[-1]
+            doc_full_cmd = match["full_cmd"]
+            doc_cmd = match["cmd"]
+            doc_hash = match["hash"]
+            flags = match["flags"]
```

Comment:
We now use groupdict, as it is more secure. The old method returned duplicated items so accesing by index was not safe.